### ISSUE TITLE
Missing demo data

### ISSUE THREAD

**Issue by hiyama341** [jonfunk21/ProteusAI#10]
First of all super cool tool and great work. :) 
I was trying to use the [demo.ipynb](https://github.com/jonfunk21/ProteusAI/blob/ce35175fceaa739e1e58b369d5f83375b33e0b66/demo/demo.ipynb) but i couldn't find the demo_data/GB1.pdb data file. 
Maybe I'm missing something but would love try out the demo.

Thanks in advance!

**Comment by jonfunk21** [comment]
Hey! Thanks for your interest in the ProteusAI!

The Jupyter notebook is a little outdated, I have removed it from the repository. I assume you are interested in a design workflow demo from what you asked. I added a design_demo.py file to the demo folder and uploaded the corresponding input file.

There are still occasionally issues with the installation of the environment, I am working on reducing the complexity of the environment to make the installation more stable, but I am on a conference next week, so it will take some time. For some people the alternative installation path also works:

```
cd proteusAI
conda env create -f proteusEnvironment.yml -y
```

I will let you know when the problems are resolved!
